### ISSUE TITLE
Theming: main nav

### ISSUE THREAD

**Issue by cbrody** [IIED-org/IIED-main#183]
Tailwind/JS/twig - priority navigation for submenu. Still to do:
- [x] Mobile menu: overlay rather than displace main div (see [related drupal.org issue](https://www.drupal.org/project/responsive_menu/issues/3274889))
- [x] Refine search box placement and size
- [x] fix for logged in / anonymous display issues
- [x] responsive search box 
- [x] adjust nav list item alignment
- [x] adjust wrapping on lg sizes (around 1020-1040px)
- [x] align search to the right on smaller screens with menu icon
- [x] adjust padding on secondary menu
- [x] issue with secondary menu order resetting to alphabetical

Carry forward to #416 
- [ ] Refine colours (code for colours: logo and secondary nav bg)
- [ ] add active menu colour/state 
- [ ] toggle button style / animation
- [ ] overlay opacity on responsive nav a little slow to appear

Toggle examples 
https://codepen.io/LarsKlopstra/pen/MWpjNzy 
https://codepen.io/duaneblake/pen/bGEgJZL
https://alpinetoolbox.com/examples/navbar

**Comment by cbrody** [comment]
Latest commit includes a search block, still needs plenty of work!

**Comment by cbrody** [comment]
Still plenty to do on this so perhaps you could take a look if you have a chance @ncktrnr ?

**Comment by cbrody** [comment]
Passing this to @ncktrnr if you have time to do some finessing?

**Comment by cbrody** [comment]
Waiting on default_content and a fix for logged in / anonymous display issues

**Comment by cbrody** [comment]
Got priority nav working again, also for logged-in users. Changes merged into dev-master and on pubs-dev.ac.iied.org.

**Comment by cbrody** [comment]
Next steps: look at dev version of responsive_menu to see if it resolves the mobile menu placement issue.

**Comment by cbrody** [comment]
offCanvas option not quite there yet but work in progress merged and deployed to pubs-dev.ac.iied.org - feedback welcome! @ncktrnr @mattjobob @DavidIIED

**Comment by ncktrnr** [comment]
Great work! Looks like the functionality is all there. The button needs to work as an open/close toggle, so we could have the menu in from right or position off canvass a little lower down. Either works. I have a list of styling changes, but let's discuss first.

**Comment by DavidIIED** [comment]
I'll second that, looking good! The display seems to skew slightly using mac/monterey/safari (browserstack) - see attached.
![pubs-dev-safari](https://user-images.githubusercontent.com/25746670/175249197-cc503b96-5f3b-4b9d-beab-e9a4765cf2e5.PNG)

**Comment by ncktrnr** [comment]
Quick note for when it comes to styling that removing the absolute positioning on secondary menu seems to make it more compatible (.priority-nav__dropdown { /* position: absolute; */ }). We don't necessarily need it to overlay on the page.

**Comment by cbrody** [comment]
Made some minor changes, merged into dev-master.

**Comment by cbrody** [comment]
I believe https://github.com/IIED-org/pubs/issues/183#issuecomment-1164091375 is now resolved, can @DavidIIED confirm? Thanks!

**Comment by cbrody** [comment]
Added a few more tweaks, now works OK.

**Comment by DavidIIED** [comment]
@cbrody , I agree the mac/monterey/safari issue resolved. Great stuff!

**Comment by cbrody** [comment]
- [ ] lg size text wrapping issue
- [ ] Move mobile menu down 60px
- [ ] white background for main menu. 
- [ ] Priority nav resize

**Comment by mattjobob** [comment]
I'm happy to consider this done @cbrody.

Just noting the unchecked boxes above. Are those elements resolved?

**Comment by cbrody** [comment]
Still a few things to do but possibly not needed at launch...
